website/components/ecosystem-integration-group/ecosystem-card/index.jsx

```diff
@@ -0,0 +1,31 @@
+import s from './style.module.css'
+import Image from '@hashicorp/react-image'
+import SmartLink from 'components/smart-link'
+
+export default function EcosystemCard({
+  companyName,
+  integrationType,
```

Comment:
nit: `type` is shorter. And might be worth looking at restricting the possible types here as well.

Reply:
Oh restrictions are def a good call.  I can throw an error on something that isn't one of the three expected, like we did with `htmlContent`/`content` in `react-text-split`